templates/security/fedramp.html

```diff
@@ -0,0 +1,296 @@
+      <hr />
+      <h2>What Canonical can provide</h2>
+    </div>
+    <div class="row--25-75">
+      <div class="col">
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <p class="p-heading--5">Patching security vulnerabilities</p>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Canonical provides 10 years of security patching for software applications and infrastructure components within the Ubuntu ecosystem. FedRAMP requires you to fix high-risk vulnerabilities within 30 days. Since starting out 20 years ago, Canonical has released patches for critical vulnerabilities within 24 hours.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row--25-75">
+      <div class="col">
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <p class="p-heading--5">FIPS-certified crypto modules</p>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Ubuntu Pro provides FIPS 140 certified cryptographic modules that are a direct drop-in replacement for the standard cryptography libraries which ship with Ubuntu by default. These libraries make it easy for you to deploy applications that use cryptographic algorithms and protocols in accordance with the FIPS 140 standard.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row--25-75">
+      <div class="col">
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <p class="p-heading--5">System hardening</p>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              The Ubuntu Security Guide allows you to harden critical systems, locking them down to approved industry standards including the CIS benchmarks and DISA STIG. This functionality enables automated auditing and remediation for the hundreds of rules specified within the hardening benchmarks.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>
+          Available on-prem
+          <br class="u-hide--small u-hide--medium" />
+          and in the cloud
+        </h2>
+      </div>
+      <div class="col">
+        <div class="u-fixed-width">
+          <div class="p-logo-section--dense">
+            <div class="p-logo-section__items">
+              <div class="p-logo-section__item">
+                <a href="/aws">
+                  <img class="p-logo-section__logo"
+                       src="https://assets.ubuntu.com/v1/5e2e7859-aws-logo.png"
+                       alt="AWS" />
+                </a>
+                <hr class="is-muted" />
+                <p>
```

Comment:
Juan's requested this in a few pages recently, I believe these should have an aria label that also includes the text in brackets on the doc for accessibility, eg:
![image](https://github.com/canonical/ubuntu.com/assets/17607612/8603a641-837c-40b6-b1bd-f78c82afff8d)

Reply:
I was wondering what that meant, thanks